cpp/include/cudf/io_functions_cpp.h

```diff
@@ -0,0 +1,29 @@
+
+/*
+ * Copyright (c) 2018, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <memory> 
+#include <arrow/io/interfaces.h>
+#include <arrow/io/file.h>
+
+//I can't add the file_handle to args because args has to be compiled
+//for cffi
+gdf_error read_csv_arrow(csv_read_arg *args,
```

Comment:
This needs Doxygen documentation https://github.com/rapidsai/cudf/blob/branch-0.6/cpp/src/examples/documentation_example.cpp